composer.json

```diff
@@ -21,7 +21,7 @@
         "symfony/yaml": "~2.3|~3|~4",
         "symfony/phpunit-bridge": "~2.7|~3|~4",
         "phpunit/phpunit": "^5.7.1|~6|~7",
-        "cucumber/cucumber": "dev-gherkin-16.0.0"
+        "cucumber/cucumber": "dev-gherkin-15.0.1"
```

Comment:
Setting it to an older version so that there is something for the workflow to do, for testing